templates/myaccount/my-orders.php

```diff
@@ -9,6 +9,8 @@
  */
 
 defined( 'ABSPATH' ) || exit;
+
+$base_url = ! empty( $_SERVER['REQUEST_URI'] ) ? esc_url( home_url( wp_unslash( $_SERVER['REQUEST_URI'] ) ) ) : admin_url(); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
why `admin_url()`?

Reply:
also, on the student dashboard, this has exactly the same problem when using permalinks without trailing slash:
![Uploading Schermata 2023-07-19 alle 09.34.51.png…]()


or using ugly permalinks (?p=124) - although we don't really care of this.

Reply:
sorry, didn't noticed the `admin_url()`, it was meant to be empty.

Fixed it. Rocco, I think the image isn't uploaded correctly, can you upload it again.

Reply:
![Schermata 2023-07-21 alle 17 15 04](https://github.com/gocodebox/lifterlms/assets/7689242/95479123-cb3d-45ca-a494-ac6095c7b307)